src/sst/core/rng/marsaglia.h

```diff
@@ -16,7 +16,7 @@
 #include <stdint.h>
 #include <sys/time.h>
 
-#include "sstrng.h"
+#include "rng.h"
```

Comment:
This change (and the corresponding change in mersenne.h) could break any code that doesn't explicitly #include sstrng.h.  If, for example, a user only included mersenne.h, but refers to SSTRandom, the code won't compile because it won't find the class SSTRandom (which is defined in sstrng.h).  I don't believe this affects anything in the elements repository, but could affect third party element libraries.  Though, many people would argue that not including sstrng.h when referring to SSTRandom, which is defined there, is bad coding practice.

Reply:
If we are to handle bad coding practices in our pull requests this is going to become quite limiting.

Reply:
Agreed.  I'm happy to leave this the way it is.  Just wanted it noted here in case it comes up later.

Reply:
I am moving to include `sstrng.h` in the random generators to address bad coding practices.